src/store/reducer/user.js

```diff
@@ -141,6 +145,18 @@ function rootReducer(state = defaultState, action) {
         users: newArray4,
         class: action.type.slice(1),
       };
+    case 'NAME':
+      const newArray8 = state.pindata.filter(
```

Comment:
Now acc. to sequence this should be `newArray5` ig.

Reply:
I have changed it to 5.